backend/src/database/entities/personnel.entity.ts

```diff
@@ -79,4 +81,31 @@ export class PersonnelEntity {
   @ManyToMany(() => TrainingEntity)
   @JoinTable({ name: 'personnel_training' })
   trainings: TrainingEntity[];
+
+  toResponseObject(): PersonnelRO {
+    return {
+      id: this.id,
+      firstName: this.firstName,
+      lastName: this.lastName,
+      email: this.email,
+      primaryPhone: this.primaryPhone,
+      secondaryPhone: this.secondaryPhone,
+      otherPhone: this.otherPhone,
+      region: this.region,
```

Comment:
Since work location is within a region, I wonder if we need both?
One region -> many locations
One person -> one location ?